extensions/blocks/eventbrite/eventbrite.php

```diff
@@ -14,6 +14,15 @@
 const FEATURE_NAME = 'eventbrite';
 const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
 
+/**
+ * Determine whether the page will be AMP.
+ *
+ * @return bool
+ */
+function is_amp_request() {
+	return class_exists( 'Jetpack_AMP_Support' ) && \Jetpack_AMP_Support::is_amp_request();
+}
```

Comment:
Good news! We now have `Blocks::is_amp_request()` to simplify things a little bit. If you rebase this PR against `master`, you should have access to it. 

For reference: #17101.

Reply:
Done in 99e2c2f